pages/about/index.md

```diff
@@ -90,7 +90,7 @@ All the information about the project's economy, the amount of the donations, th
 [**press@asyncapi.io**](mailto:press@asyncapi.io).
 - [**Twitter**](https://twitter.com/asyncapispec?lang=es): It's used for disseminating all news and information regarding the project, resolving user queries, and sharing the latest updates within AsyncAPI.
 - [**Slack workspace**](https://www.asyncapi.com/slack-invite): The main place of encounter for the whole community of AsyncAPI.
-- [**Newsletter**](https://www.asyncapi.com/subscribe/): To learn about the status of the project, news, actualizations, recorded events, and meetings.
+- [**Newsletter**](https://www.asyncapi.com/newsletter): To learn about the status of the project, news, actualizations, recorded events, and meetings.
 - [**Blog**](/blog): It’s used to disseminate information related to the project. It's also open to collaborations.
```

Comment:
```suggestion
- [**Blog**](https://www.asyncapi.com/blog): It’s used to disseminate information related to the project. It's also open to collaborations.
```

this link doesn't work, changing it to website blog link

Reply:
It works for me. 🤔 can you check if it works on the website? `https://www.asyncapi.com` will automatically be added to all the links that start with `/` on the website.

Reply:
https://github.com/KhudaDad414/website/actions/runs/3436567899/jobs/5730231016